<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>446</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>755</width>
    <height>418</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(254, 254, 254);
font: 75 13pt &quot;Inter&quot;;
border-radius: 20px;
	

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>803</width>
       <height>100</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="slogan">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>13</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Editar senha&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>270</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>803</width>
       <height>270</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="Input_appName">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>9</pointsize>
        <weight>25</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border: 1px solid  rgb(127, 127, 127); 
	border-radius: 5px ; 
	background-color: rgb(254, 254, 254);
	font: 200 9pt &quot;Inter&quot;;
}

QLineEdit:hover{
	border:1px solid  rgb(254, 254, 254); 
}

QLineEdit:focus{
	color: rgb(000, 000, 000);
	border:1px solid  rgb(14, 78, 20); 
}</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="TextoName">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>13</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:450;&quot;&gt;Nome da &lt;br/&gt;ferramenta&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="TextoName_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>13</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:450;&quot;&gt;Usu√°rio&lt;br/&gt;de Acesso&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Input_appUsername">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border: 1px solid  rgb(127, 127, 127); 
	border-radius: 5px ; 
	background-color: rgb(254, 254, 254);
	font: 200 9pt &quot;Inter&quot;;
}

QLineEdit:hover{
	border:1px solid  rgb(254, 254, 254); 
}

QLineEdit:focus{
	color: rgb(000, 000, 000);
	border:1px solid  rgb(14, 78, 20); 
}</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="TextoName_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>40</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>13</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:400;&quot;&gt;Senha&lt;br/&gt; de Acesso&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Input_appPSWRD">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>40</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>9</pointsize>
        <weight>25</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	border: 1px solid  rgb(127, 127, 127); 
	border-radius: 5px ; 
	background-color: rgb(254, 254, 254);
	font: 200 9pt &quot;Inter&quot;;	
}

QLineEdit:hover{
	border:1px solid  rgb(254, 254, 254); 
}

QLineEdit:focus{
	color: rgb(000, 000, 000);
	border:1px solid  rgb(14, 78, 20); 
}</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_generate">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>100</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>9</pointsize>
        <weight>25</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">	font: 200 9pt &quot;Inter&quot;;</string>
      </property>
      <property name="text">
       <string>Gerar senha?</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_symbol">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>130</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	font: 200 9pt &quot;Inter&quot;;</string>
      </property>
      <property name="text">
       <string>Simbolos (!, $, #, %)</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_uppercase">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>160</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	font: 200 9pt &quot;Inter&quot;;</string>
      </property>
      <property name="text">
       <string>Mai√∫sculos</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_caracter">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>100</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	font: 200 9pt &quot;Inter&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="TextoName_Name_4">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>100</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>9</pointsize>
        <weight>25</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">	font: 200 9pt &quot;Inter&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:400;&quot;&gt;Caracteres&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_number">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>130</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	font: 200 9pt &quot;Inter&quot;;</string>
      </property>
      <property name="text">
       <string>N√∫meros</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_generate">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>170</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(25, 144, 35);
	font: 200 12pt &quot;Inter&quot;;
	color: rgb(255, 255, 255);
	border-radius: 5px;
}

QPushButton:hover{
	background-color: rgb(28, 162, 39);
	font: 200 12pt &quot;Inter&quot;;
	color: rgb(255, 255, 255);
	border-radius: 5px;
}

QPushButton:hover:pressed{
	background-color: rgb(24, 152, 35);
	font: 200 12pt &quot;Inter&quot;;
	color: rgb(255, 255, 255);
	border-radius: 5px;
}

QPushButton:disabled {
	background-color: rgb(102, 132, 105);
}

</string>
      </property>
      <property name="text">
       <string>Gerar</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="maximumSize">
      <size>
       <width>803</width>
       <height>120</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton_save">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>0</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>12</pointsize>
        <weight>25</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(25, 144, 35);
	font: 200 12pt &quot;Inter&quot;;
	color: rgb(255, 255, 255);
	border-radius: 5px;
}

QPushButton:hover{
	background-color: rgb(28, 162, 39);
	font: 200 12pt &quot;Inter&quot;;
	color: rgb(255, 255, 255);
	border-radius: 5px;
}

QPushButton:hover:pressed{
	background-color: rgb(24, 152, 35);
	font: 200 12pt &quot;Inter&quot;;
	color: rgb(255, 255, 255);
	border-radius: 5px;
}

</string>
      </property>
      <property name="text">
       <string>Salvar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_erase">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>12</pointsize>
        <weight>25</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(255, 26, 26);
	font: 200 12pt &quot;Inter&quot;;
	color: rgb(255, 255, 255);
	border-radius: 5px;
}

QPushButton:hover{
	background-color: rgb(175, 17, 17);
	font: 200 12pt &quot;Inter&quot;;
	color: rgb(255, 255, 255);
	border-radius: 5px;
}

QPushButton:hover:pressed{
	background-color: rgb(255, 105, 107)
	font: 200 12pt &quot;Inter&quot;;
	color: rgb(255, 255, 255);
	border-radius: 5px;
}

</string>
      </property>
      <property name="text">
       <string>Apagar</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
